housekeeper/store/filters/archive_filters.py

```diff
@@ -2,14 +2,17 @@
 from enum import Enum
 from typing import Callable
 
-from sqlalchemy import and_
+from sqlalchemy import and_, not_
 from sqlalchemy.orm import Query
 from sqlalchemy.sql.elements import BooleanClauseList
 
 from housekeeper.store.models import Archive
 
 ARCHIVING_ONGOING: BooleanClauseList = and_(Archive.archiving_task_id, Archive.archived_at == None)
 RETRIEVAL_ONGOING: BooleanClauseList = and_(Archive.retrieval_task_id, Archive.retrieved_at == None)
+RETRIEVAL_NOT_ONGOING: BooleanClauseList = not_(
+    and_(Archive.retrieval_task_id != None, Archive.retrieved_at == None)
+)
```

Comment:
```suggestion
RETRIEVAL_NOT_ONGOING: BooleanClauseList = not_(RETRIEVAL_ONGOING)
```
might be easier to read but maybe there is a reason the check with None needs to be explicit in this case?